skbeam/core/mask.py

```diff
@@ -108,3 +109,84 @@ def threshold_mask(images, threshold, mask=None):
         if len(bad_pixels[0]) != 0:
             mask[bad_pixels] = 0
         yield mask
+
+
+def margin_mask(img_shape, edge_size):
+    """
+    Mask the edge of an image
+
+    Parameters
+    -----------
+    img_shape: tuple
+        The shape of the image
+    edge_size: int
+        Number of pixels to mask from the edge
+
+    Returns
+    --------
+    2darray:
+        The mask array, bad pixels are 0
+    """
+    mask = np.zeros(img_shape, dtype=bool)
+    mask[edge_size:-edge_size, edge_size:-edge_size] = True
+    return mask
+
+
+def ring_blur_mask(img, r, alpha, bins, mask=None):
```

Comment:
This function does not depend on rings, does no blurring, and generates, not applies a mask.  That is like 0.5 / 3 for words in the name being related to what it does.